SOURCES/systemd-219-Backport-udev-exclude-DRBD-from-block-device-ownersh.XCP-ng.patch

```diff
@@ -0,0 +1,56 @@
+From a4621ae866977991eed65d143ffec525659c9a1b Mon Sep 17 00:00:00 2001
+From: Ronan Abhamon <ronan.abhamon@vates.fr>
```

Comment:
I would prefer the raw patch from upstream, with a comment on top that explains where you got it from and what changes you had to do for it to apply, if necessary.

Patch name would also be based on the upstream commit message, and end in .backport.patch rather than .XCP-ng.patch, according to old naming rules buried in the forum :)